### PR TITLE
replace mariadb driver with mysql driver

### PR DESCRIPTION
**Pull request by erizocosmico**
Fixes https://github.com/src-d/gitbase/issues/807

### CODE REVIEW

**Comment by erizocosmico** [comment]
Do not merge yet, there are a couple issues

**Comment by erizocosmico** [comment]
Fixed. It turns out for the MySQL driver JSON is StringType rather than BinaryType so it had to be overwritten in our dialect.

File: src/test/scala/tech/sourced/gitbase/spark/BaseGitbaseSpec.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@ trait BaseGitbaseSpec extends FlatSpec with Matchers with BeforeAndAfterAll with
 
   private val resourcePath = Paths.get(getClass.getResource("/").toString)
 
-  private val gitbaseVersion = "v0.20.0-beta3"
+  private val gitbaseVersion = "v0.20.0-beta4"
```
Comment (ajnavarro) [inline src/test/scala/tech/sourced/gitbase/spark/BaseGitbaseSpec.scala]: shall we change this on all the places?

Comment (erizocosmico) [reply src/test/scala/tech/sourced/gitbase/spark/BaseGitbaseSpec.scala]: Changed. Forgot to change it in README and docker-compose

**Pull request merged**